vm/vmimpl/console.go

```diff
@@ -134,6 +135,25 @@ func OpenAdbConsole(bin, dev string) (rc io.ReadCloser, err error) {
 	return OpenRemoteConsole(bin, "-s", dev, "shell")
 }
 
+// Open console log by cmd.
+func OpenConsoleByCmd(cmdline string) (rc io.ReadCloser, cmd *exec.Cmd, err error) {
```

Comment:
Please note that other similar methods don't return `exec.Cmd` explicitly -- the process is killed once `rc.Close()` is called. There's no reason not to follow the approach here as well.